src/main/java/org/example/backendspringbootdomainmodelsassignment/dtos/TelevisionDto.java

```diff
@@ -0,0 +1,34 @@
+package org.example.backendspringbootdomainmodelsassignment.dtos;
+
+import jakarta.validation.constraints.NotBlank;
+
+public class TelevisionDto {
+    public long id;
+
+    @NotBlank
+    public String type;
+
+    @NotBlank
+    public String brand;
+
+    @NotBlank
+    public String name;
+
+    public double price;
+    public double available_size;
+    public int refresh_rate;
+
+    @NotBlank
+    public String screen_type;                                                                                          // For some reason when using camelCase the input functions (save & update)
```

Comment:
Waarschijnlijk ligt dit aan je input, snakeCase is niet heel netjes om te gebruiken.